src/frontend/src/pages/CalendarPage/DesignReviewDetailPage/UserAvailabilitesView.tsx

```diff
@@ -85,19 +93,22 @@ const UserAvailabilites: React.FC<UserAvailabilitiesProps> = ({
                 width: '10vw'
               }}
             >
-              Unvailable
+              Unavailable
             </Typography>
-            {currentUnavailableUsers.map((user) => (
-              <Typography style={{ textAlign: 'center', fontSize }}>{fullNamePipe(user)}</Typography>
-            ))}
+            <Box sx={{ maxHeight: '350px', overflowY: 'auto' }}>
+              {currentUnavailableUsers.map((user) => (
+                <Typography style={{ textAlign: 'center', fontSize }}>{fullNamePipe(user)}</Typography>
+              ))}
+            </Box>
           </Box>
         </Box>
         <Box
           style={{
             marginTop: 'auto',
             display: 'flex',
             justifyContent: 'flex-end',
-            gap: '10px'
+            gap: '10px',
+            overflow: 'auto'
```

Comment:
If you change this from auto to scroll, does the scroll bar show up automatically?